readme.md

```diff
@@ -169,17 +187,35 @@ Inside `app/views/artist/index.html.erb`:
 Just put: `<h1>All Artists</h1>`
 
 Great, now let's refresh the page and there should be no more errors so we know everything has been wired up correctly!
+---
 
 However, let's think about what we really want to see when we visit this page. When we visit `/artists`, we expect to see information about all artists!
 
 So now the question is how can we dynamically generate a view for all of our data about artists?
 
-### Instance Variables
-Recall from our Sinatra lessons how we used instance variables in our controllers.
+**(STWG)**: What does it mean to have our view update dynamically?
+
+> Means we can write abstractive code that will update and render the appropriate data for all the records in our query. Can you imagine if you were a store with 5000 products, having to write out or hard code all those listings?
+
+Lets look at [Amazon](http://amazon.com):
+- What do you think happens behind the scenes anytime some one makes a search, say for `Ruby on Rails books`?
+- What parts of search result are similar across each result?
+- What components do you notice change instance by instance?
+- How would you potential model the data you are seeing? What are a listings attributes?
```

Comment:
I like to use the example of Facebook: "Facebook has like 4 billion users. They definitely do not have 4 billion profile HTML pages lying around -- that would be a ridiculous amount of data. Instead, they have ONE HTML page lying around, and when you look at someone's profile they insert the appropriate data into the appropriate places in that HTML, and send your computer the result."